samples/security/vulnerability findings/slack_notifications_for_critical_vulnerabilities.yaml

```diff
@@ -0,0 +1,232 @@
+metadata:
+  version: "1"
+  dependencies:
+    apps:
+      - id: dynatrace.automations
+        version: ^1.728.0
+      - id: dynatrace.slack
+        version: ^2.0.2
```

Comment:
same: version is not released, but at least everything seems to work after importing the WF in prod